api/views/people.py

```diff
@@ -0,0 +1,44 @@
+from api.share.tools import build_correct_http_response
+from api.share.requests.users import GetUsersCountRequest
+from api.share.requests import InvalidRequestObject
+from api.use_cases.get_users_count import UsersCount
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from http import HTTPStatus
+
+
+class UserCountView(APIView):
+
+    def get(self, request):
+        try:
+            params = request.query_params
+            request_object = GetUsersCountRequest.from_dict(params) # object contains correct parameters
+            # if something went wrong
```

Comment:
what's more, you have "if something went wrong" inside a `try` block. That is another code smell to me

Reply:
This is comment for the next line of code: _if isinstance(request_object, InvalidRequestObject):_. This construction allows to handle errors inside user cases and not raise them to this level. In particular because of this we can remove try..expected block (see previous comment).

Reply:
I know, but it looks like you put something into `try` and then if something goes wrong you don't call `except` but continue within `try`. That's just how it looks for an outside reader :)

Reply:
See previous comment